### ISSUE TITLE
Open channel remote balance clarification

### ISSUE THREAD

**Issue by cpunk2140** [LN-Juggernaut/juggernaut-desktop#55]
Since users have the possibility to set the remote balance, I think Juggernaut should make it extra clear of what remote balance means, otherwise users might not realising that they're actually sending sats to the other side when doing this.

Maybe it would actually make sense at a point in the future to not show some things in the UI, like the remote balance and target confirmations, unless the user enables an "Advanced" mode in settings.

**Comment by johncantrell97** [comment]
Yeah, I think it's really important to push a few sats (1000 default) when opening a channel with messaging in mind.  If you do not push any sats then you cannot receive any messages.  1000 sats is small enough where it doesn't make sense for the remote node to close the channel to take the 1000 sats because most of it will be lost to on-chain fees.

**Comment by cpunk2140** [comment]
I agree, 1000 sats is a good default and it's necessary.

The problem is that as the field is adjustable, and it's a source of confusion for people new to ⚡️, an user might set a considerable larger amount (say, for example, half of the channel capacity) thinking that this would give him considerably more inbound liquidity, but not realising that he's actually sending the sats.

I think that's why other wallets chose not to show the field in the UI, while still having a 1000 sats default. If you want to have more than 1000, you can still have it, but you need to make an explicit send after opening the channel, which is not confusing even for people new to ⚡️.
